source/fundamentals/periodic-executors.txt

```diff
@@ -65,52 +76,42 @@ callback to terminate itself soon after the topology is freed.
 
 Solid lines represent strong references, dashed lines weak ones:
 
-.. generated with graphviz: "dot -Tpng periodic-executor-refs.dot > periodic-executor-refs.png"
-
 .. image:: ../images/periodic-executor-refs.png
    :alt: Periodic executor references
 
-See Stopping Executors above for an explanation of the ``_EXECUTORS`` set.
-
-It is a requirement of the `Server Discovery And Monitoring Spec`_ that a
-sleeping monitor can be awakened early. Aside from infrequent wakeups to do
+Aside from infrequent wakeups to do
 their appointed chores, and occasional interruptions, periodic executors also
 wake periodically to check if they should terminate.
 
-Our first implementation of this idea was the obvious one: use the Python
-standard library's threading.Condition.wait with a timeout. Another thread
-wakes the executor early by signaling the condition variable.
+Initially, {+driver-short+} used the Python
+standard library's ``threading.Condition.wait()`` method with a timeout. Another thread
+woke the executor early by signaling the condition variable.
 
 A topology cannot signal the condition variable to tell the executor to
 terminate, because it would risk a deadlock in the garbage collector: no
 destructor or weakref callback can take a lock to signal the condition variable
-(see `PYTHON-863`_); thus the only way for a dying object to terminate a
-periodic executor is to set its "stopped" flag and let the executor see the
-flag next time it wakes.
+(see `PYTHON-863 <https://jira.mongodb.org/browse/PYTHON-863>`__). The only way for a
+dying object to terminate a periodic executor is to set its "stopped" flag and let the
+executor see the flag next time it wakes.
 
 We erred on the side of prompt cleanup, and set the check interval at 100ms. We
 assumed that checking a flag and going back to sleep 10 times a second was
 cheap on modern machines.
 
-Starting in Python 3.2, the builtin C implementation of lock.acquire takes a
-timeout parameter, so Python 3.2+ Condition variables sleep simply by calling
-lock.acquire; they are implemented as efficiently as expected.
+Starting in Python 3.2, the built-in C implementation of the ``lock.acquire()`` method
+takes a ``timeout`` parameter, so Python 3.2+ Condition variables sleep simply by calling
```

Comment:
Not sure if Condition should be lowercased, or if it's a keyword that should be monospaced?

```suggestion
takes a ``timeout`` parameter, so Python 3.2+ condition variables sleep simply by calling
```

Reply:
yeah, should be lowercase